site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -60,27 +61,58 @@ The application provides a user interface which allows the consumer of the appli
 
 The first solution is not really what we want to be doing because the consumer will potentially have to create objects on their Snowflake instance just to satisfy the application's requirements, so this Quickstart will deliver the second solution.
 
+> aside positive
+> 
+> **Note** - The following steps explain how to create the app using the Snowflake CLI, as this option presents as a faster, more straightforward way to build Native Apps. You can also accomplish it by manually creating the folder structure and executing the SQL commands in Snowsight, but this alternative is out of the scope for this Quickstart.
+
+The files explained in the following steps, are already present in the project folder, and it is merely a deeper explanation of its contents.
+
+
+<!-- ------------------------ -->
+## Snowflake CLI First Steps
+Duration: 7
+
+The datamapping native app project resides in a Snowflake repository which contains several app examples for the user to learn its way into Snowflake Native Apps.
+
+The instructions on cloning the project repository and its execution are given in the readme file of this github [link NEEDS UPDATE AFTER PR](https://github.com/snowflakedb/native-apps-examples).
+
+### Clone instructions 
+
+1. Clone the project repo in your local machine. Repo could be cloned running:
+    ```sh
+    snow app init --template-repo git@github.com:snowflakedb/native-apps-examples.git --template datamapping-app
+    ```
+2. Replace the CSV filepath inside the **prepare_data.sh** file, line 28, like this:  
+    ```sh
+    snow object stage copy /USER_PATH_HERE/IP2LOCATION-LITE-DB11.CSV @location_data_stage --database ip2location --schema ip2location
```

Comment:
Use `snow stage copy` and latest version of Snowflake CLI as the `snow object stage` commands are going to be deprecated in the next release.